extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -552,7 +553,14 @@ public void handle(HttpServerRequest event) {
             }
             AccessLogHandler handler = new AccessLogHandler(receiver, accessLog.pattern, getClass().getClassLoader(),
                     accessLog.excludePattern);
-            httpRouteRouter.route().order(Integer.MIN_VALUE).handler(handler);
+            if (nonApplicationDedicatedRouter && !nonApplicationAttachedToMainRouter) {
```

Comment:
We're already _almost_ drowning in config options here, but should there be an option for suppressing non-application endpoint access logs when attached to a dedicated endpoint? (for metrics, e.g., we have ways to ignore requests.. should you be able to ignore the non-application endpoint?)

Reply:
hmm, can't that be done already with `quarkus.http.access-log.exclude-pattern`?

Reply:
I've managed a quick confirmation with Stuart.. and we should be able to move both handlers to the main router, such that it cascades universally (there is another issue open in essentially the same space).

Reply:
personally, I think that it makes sense since `quarkus.http.non-application-root-path` is also under `quarkus.http.*`

Reply:
basically, everything is still HTTP, and we don't make the distinction (from user's point of view) for non-app except for the option to put it elsewhere. At least from what I can find.

Reply:
I like your answers. Thank you. ;)

Reply:
@ebullient so are we good with this one?